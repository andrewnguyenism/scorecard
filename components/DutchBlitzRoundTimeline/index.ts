export { DutchBlitzRoundTimeline } from './DutchBlitzRoundTimeline'

export type { DutchBlitzRound } from './DutchBlitzRoundTimeline'
