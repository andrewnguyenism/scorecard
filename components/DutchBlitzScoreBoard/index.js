export {
  DutchBlitzScoreBoardHeader,
  DutchBlitzScoreBoardPlayer,
  DutchBlitzScoreBoard,
} from "./DutchBlitzScoreBoard";
