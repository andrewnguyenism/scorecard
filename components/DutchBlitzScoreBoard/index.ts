export { DutchBlitzScoreBoard } from './DutchBlitzScoreBoard'
export { DutchBlitzScoreBoardHeader } from './DutchBlitzScoreBoardHeader'
export { DutchBlitzScoreBoardPlayer } from './DutchBlitzScoreBoardPlayer'
