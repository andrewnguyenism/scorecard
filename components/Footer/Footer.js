export const Footer = () => (
  <div className="text-center text-xs py-4 mb-4">
    Created by Andrew Nguyen. v1-beta
  </div>
)