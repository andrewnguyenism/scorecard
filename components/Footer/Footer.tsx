import Link from "next/link";

export const Footer = () => (
  <div className="text-center text-xs py-4 mb-4">
    <Link href="/changelog"><a className="block mb-2 underline">v2</a></Link>
    <p>Created by Andrew Nguyen in 2020.</p>
  </div>
)