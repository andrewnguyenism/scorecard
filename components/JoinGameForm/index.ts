export { JoinGameFormError, JoinGameForm } from './JoinGameForm'
