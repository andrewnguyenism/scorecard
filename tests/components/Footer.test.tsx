import { render } from '../test-utils'

import { Footer } from '@/components/Footer'

describe('Footer component', () => {
  it('renders component', () => {
    const { asFragment } = render(<Footer />)
    expect(asFragment()).toMatchSnapshot()
  })
})
