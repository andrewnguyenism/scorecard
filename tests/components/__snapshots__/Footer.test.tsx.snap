// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders component 1`] = `
<DocumentFragment>
  <div
    class="text-center text-xs py-4 mb-4"
  >
    <a
      class="block mb-2 underline"
      href="/changelog"
    >
      v3
    </a>
    <p>
      Created by Andrew Nguyen in 2020.
    </p>
  </div>
</DocumentFragment>
`;
